scripts/zones/Abyssea-Grauberg/IDs.lua

```diff
@@ -71,6 +71,28 @@ zones[xi.zone.ABYSSEA_GRAUBERG] =
     },
     mob =
     {
+        IRONCLAD_SUNDERER            = 17818041,
```

Comment:
These could probably be moved to dynamic lookups, but if so, the ID local in those scripts would need to pull data from the zones global table

Reply:
The idea was to convert the closed PRs first and then get back to this. (Hopefully I will have gotten the hang of the dynamic lookup by then)